NSS-Day-Cohort-53/tabloidcli-bubbling-bullfrogs
# completed NoteManger.cs

#### Changes Made
1. Added file `NoteManager.cs` to `UserInterfaceManagers` directory.
2. Filled out file
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout NoteManager
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    Nav into post management (4), select note management, select any number and it should list notes
